src/features/task/taskSlice.js

```diff
@@ -7,7 +7,12 @@ const initialState = {
 const taskSlice = createSlice({
     name: "task",
     initialState,
-    reducers: {},
+    reducers: {
+        addTask: (state, action) => {
+            state.tasks.unshift(action.payload);
```

Comment:
using `unshift` for showing recent tasks at the top right?

Reply:
yes, bhaia. So that I don't need to reverse it always to show the recent tasks at the top. Thank you!